### DISCUSSION TITLE
More websites microsoft edge

### DISCUSSION THREAD

**Discussion by mustuthebest** [blueedgetechno/win11React#229] [Q&A] [answered]
I would like to be able to visit every website on microsoft edge/bing. Many refuse to coonect. also some new apps and features would be nice

**Comment by Ccode-lang** [comment, marked as answer]
Iframe does not work with all websites for security reasons. Getting all websites in browser is impossible for this reason.

**Comment by Ccode-lang** [comment]
Bypass of iframe can cause companies to be mad.

**Reply by blueedgetechno** [reply]
And I mean, real mad. They will track down our addresses and hire a sniper on us.
